src/main/resources/db/migration/all/20240610154305__create_cas1_out_of_service_bed_details_table.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE cas1_out_of_service_bed_revisions (
```

Comment:
nice, I agree that "revisions" work well here. Do you want to rename the migration to reflect the the new name?